doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -21,6 +21,47 @@ This specification is licensed under the
 [FIWARE Open Specification License]
 (http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/FI-WARE_Open_Specification_Legal_Notice_%28essential_patents_license%29).
 
+## FIWARE NGSI information and data model introduction
+
+The FIWARE NGSI (Next Generation Service Interface) API defines 
+
+* a **data model** for context information, based on a simple information model using the notion of *context entities*.
+* a **context data interface** for exchanging information by means of query, subscription, and update operations.
+* a **context availability interface** for exchanging information on how to obtain context information.
+
+### Context Entities
+
+Context entities, or simply called entities, are the center of gravity in the FIWARE NGSI information model. An entity represents a real-world object like e.g. a sensor, a person, or a room. Each entity is uniquely identified by its **entity id**. 
+
+Furthermore, the type system of FIWARE NGSI enables entities to have an **entity type**. Entity types are semantic types; they are intended to describe the type of real-world object represented by the entity. For example, a context entity with id *sensor-365* could have the type *temperatureSensor*.
+
+### Context Elements
```

Comment:
Is is really important the "context element" concept in the API? We have found that use to be difficult to understand by users. On the contrary, entities, attirbutes and metadata are concepts which they find easy to understand. Maybe the best way of defining a context element is as entity + subsect of its attributes. Is that interpretation correct?